src/main/java/org/ngafid/accounts/EmailType.java

```diff
@@ -98,6 +101,16 @@ public static EmailType[] getAllTypes() {
         return values();
     }
 
+    public static String[] getEmailTypeKeysRecent(Boolean doRefresh) {
```

Comment:
In java `boolean` is the primitive boolean type and `Boolean` is a boxed version (i.e. it is heap allocated as an object which contains a `boolean`)